ui/components-react/src/components-react/properties/renderers/value/MultilineTextPropertyValueRenderer.tsx

```diff
@@ -64,19 +62,21 @@ interface MultilineTextRenderer {
 
 /** @internal */
 export const MultilineTextRenderer: React.FC<MultilineTextRenderer> = (props) => { // eslint-disable-line @typescript-eslint/no-redeclare
-  const { rerenderContext, numRerenders, rerender } = useRerender({ contentOverflows: false });
   const spanRef = useRef<HTMLSpanElement>(null);
   const divRef = useRef<HTMLDivElement>(null);
   const previousHeightRef = useRef(0);
+  const contentOverflows = useRef<boolean>(false);
+  const [numRerenders, setNumRerenders] = React.useState<number>(0);
```

Comment:
Is there a need to count numRerenders? I think it would be enough to put `contentOverflows` in state instead of ref.

Reply:
fixed